stages/prompt: add ability to set separate labels and values for choices (#16693)
goauthentik/authentik: website/docs/add-secure-apps/flows-stages/stages/prompt/index.md

By default, the placeholder is interpreted as-is. If you enable _Interpret placeholder as expression_, the placeholder
will be evaluated as a Python expression. This happens in the same environment as [_Policies_](../../../../customize/policies/expression.mdx).

In the case of `Radio Button Group` and `Dropdown` prompts, this field defines all possible values (choices). When interpreted as-is, only one value will be allowed (the placeholder string). When interpreted as expression, a list of values can be returned to define multiple choices. For example, `return ["first option", 42, {"label": "another option", "value": "some value"}]` defines 3 possible values.

> A choice can be represented as a string or other primitive (which will be turned into a string), or as an object with parameters `value` and/or `label`.Sow `return ["Option 1"]` is equivalent to `return [{"label": "Option 1", "value": "Option 1"}]`.

Reviewer: just a minor typo:  
```suggestion
A choice can be represented as a string or other primitive (which will be turned into a string), or as an object with parameters `value` and/or `label`. So `return ["Option 1"]` is equivalent to `return [{"label": "Option 1", "value": "Option 1"}]`.
```

> A choice can be represented as a string or other primitive (which will be turned into a string), or as an object with parameters `value` and/or `label`. So `return ["Option 1"]` is equivalent to `return [{"label": "Option 1", "value": "Option 1"}]`.

Reviewer 2: ```suggestion
A choice can be a string (or any primitive that will be converted to a string) or an object with `value` and/or `label` properties. For example, `return ["Option 1"]` is equivalent to `return [{"label": "Option 1", "value": "Option 1"}]`.
```
